src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/DisplayItemIdsOnItemsCapabilityProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+#if DEBUG
+
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem;
+
+namespace Microsoft.VisualStudio.Packaging
+{
+    [Export(ExportContractNames.Scopes.ProjectService, typeof(IProjectCapabilitiesProvider))]    
+    [AppliesTo(ProjectCapabilities.AlwaysApplicable)]
+    internal class DisplayItemIdsOnItemsCapabilityProvider : ProjectCapabilitiesProviderBase
```

Comment:
Does this type add `CPS_DisplayItemIdOnProjectNodes` and `DiagnoseCapabilities` to all projects unconditionally?